i18n/miraheze/es.json

```diff
@@ -2,30 +2,54 @@
 	"@metadata": {
 		"authors": [
 			"Wynter"
+			"AlvaroMolina"
 		]
 	},
 	"centralauth-groupname": "Miraheze",
 	"mirahezemagic-description": "Mensajes de MediaWiki, scripts y ganchos específicos para Miraheze",
+	"miraheze-categories": "Categorías",
+	"miraheze-donate": "Donar",
+	"miraheze-helpcenter": "Acerca de Miraheze",
+	"miraheze-requestfeatures": "Solicitar características para su wiki",
+	"miraheze-requestwiki": "Solicitar una wiki",
+	"miraheze-warnmessage": "<div class=\"wikitable\" style=\"text-align: center; width: 90%; margin-left: auto; margin-right:auto; padding: 15px; border: 4px solid black; background-color: #EEE;\"> <span class=\"plainlinks\">El [https://meta.miraheze.org/wiki/Stewards%27_noticeboard '''Staff de Miraheze'''] ha notado que este wiki [[Special:RecentChanges|'''no tiene ediciones''']] o [[Special:Logs|'''acciones registradas''']] en los últimos 45 días. Si desea evitar que este wiki sea '''cerrado''', por favor comience a mostrar signos de actividad aquí. Si no hay señales de que este wiki se use dentro de los próximos 15 días, este wiki puede ser cerrado por la [https://meta.miraheze.org/wiki/Dormancy_Policy '''Política de Inactividad''']. Esta wiki será elegible para su adopción por otro usuario. Si no se adopta y sigue inactivo en 135 días a partir de ahora, esta wiki será elegible para ser '''borrada'''. Por favor asegúrese de familiarizarse con la [https://meta.miraheze.org/wiki/Dormancy_Policy Política de Inactividad] de Miraheze. Si hay actividad en este wiki, puedes continuar y remover de [[MediaWiki:Sitenotice]] este aviso. Si tiene alguna otra pregunta o inquietud, no dude en [https://meta.miraheze.org/wiki/Stewards%27_noticeboard '''contactarnos'''].</span></div>",
 	"group-donator": "Donadores",
-	"group-donator-member": "Donador",
+	"group-donator-member": "donador",
 	"grouppage-donator": "m:Finance",
 	"group-steward": "Stewards",
 	"group-steward-member": "steward",
 	"grouppage-steward": "m:Stewards",
 	"group-sysadmin": "Administradores de sistema",
 	"group-sysadmin-member": "administrador de sistema",
 	"grouppage-sysadmin": "m:System administrators",
-	"group-wikicreator": "Creadores de Wiki",
-	"group-wikicreator-member": "creador de wiki",
+	"group-oversight": "Supresores",
+	"group-oversight-member": "supresor",
+	"grouppage-oversight": "m:Oversight",
+	"group-wikicreator": "Creadores de Wikis",
+	"group-wikicreator-member": "creador de wikis",
+	"group-rollbacker": "Reversores",
+	"group-rollbacker-member": "reversor",
+	"grouppage-rollbacker": "m:Meta:Rollbacker",
+	"group-autopatrolled": "Autoverificados",
+	"group-autopatrolled-member": "autoverificado",
+	"grouppage-autopatrolled": "m:Meta:Autopatrolled",
+	"group-confirmed": "Confirmados",
+	"group-confirmed-member": "confirmado",
 	"grouppage-wikicreator": "m:Wiki creators",
-	"protect-level-user": "Permitir solo los usuarios registrados",
+	"protect-level-user": "Solo usuarios registrados",
```

Comment:
The english version is "Allow only registered users" so please change it back to "Permitir"